bedrock/firefox/templates/firefox/privacy_tour/index.html

```diff
@@ -11,6 +11,20 @@
 
 {% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
 
+{% block page_title %}
```

Comment:
Come to think of it, would the `firefox/independent` page be a better fit for this, since that was the publically viewable campaign page for this release?